src/mixins/text.js

```diff
@@ -171,129 +168,165 @@ export const text = {
             
         },
         about_p6() {
-            return this.$store.state.spanishState == true ? 'PLACEHOLDER FOR SPANISH TEXT' : 'For site specific information please contact the local USGS Water Science Center. For other inquiries, please contact our team at';
+            return this.$store.state.spanishState == true ? 'Para obtener información específica del lugar, por favor póngase en contacto con el Centro local de Ciencias del Agua del USGS. Para otras consultas, por favor póngase en contacto con nuestro equipo en:' : 'For site specific information please contact the local USGS Water Science Center. For other inquiries, please contact our team at';
             
         },
-        about_p6_link() {
-            return this.$store.state.spanishState == true ? 'PLACEHOLDER FOR SPANISH TEXT' : 'gs_real_time_flood_impact_map_group@usgs.gov.';
-            
+
+        activeFlooding() {
+            return this.$store.state.spanishState == true ? 'Inundación activa' : 'Active Flooding';
         },
-        femaBFE() {
-            return this.$store.state.spanishState == true ? 'PLACEHOLDER FOR SPANISH TEXT' : 'FEMA 100-year BFE';
+
+        reset(){
+            let element = document.getElementById('resetbutton');
+            if(this.$store.state.spanishState == true){
+                element.innerHTML = 'Restablecer la vista'
+            }
+            else{
+                element.innerHTML = 'Reset View'
+            }
         },
-        activeFlooding() {
-            return this.$store.state.spanishState == true ? 'PLACEHOLDER FOR SPANISH TEXT' : 'Active Flooding';
+        siteLayers() {
+            return this.$store.state.spanishState == true ? 'Capas de información del sitio' : 'Site Layers ';
         },
+        activeLayers() {
+            return this.$store.state.spanishState == true ? 'Capas de información de inundación activa' : 'Active Flooding Layers';
+        },
+        showActive() {
+            return this.$store.state.spanishState == true ? 'Mostrar todas las capas de información de inundación activa' : 'Show all Active flooding Layers';
+        },
+        date() {

```

Comment:
The `date` method conflicts with a model of the same name which is generating that error in the console. So lets change the name of the method in the mixin.

```suggestion
        dateText() {

```